compiler/sc2.cpp

```diff
@@ -388,6 +389,9 @@ static void readline(unsigned char *line)
       line+=strlen((char*)line);
     } /* if */
     fline+=1;
+    sym=findconst("__line",NULL);
+    assert(sym!=NULL);
+    sym->addr_=fline;

```

Comment:
You should probably do `sym->setAddr(fline);` instead to be consistent.

Reply:
Thanks.